library/globals.inc.php

```diff
@@ -2719,6 +2712,45 @@ function gblTimeZones()
             xl('Enable OpenEMR RESTful API. SSL Required')
         ),
 
+        'payment_gateway' => array(
+            xl('Select Credit Card Payment Gateway'),
+            array(
+                'InHouse' => xl('In House'),
+                'AuthorizeNet' => xl('Authorize.Net'),
+                'Stripe' => xl('Stripe')
+            ),
+            'InHouse',
+            xl('Enable a Payment Gateway Service for processing credit card transactions')
+        ),
+
+        'gateway_mode_production' => array(
+            xl('Set Gateway to Production Mode'),
+            'bool',                           // data type
+            '0',
+            xl('Check this to go live. Not checked is testing mode.')
+        ),
+
+        'gateway_public_key' => array(
+            xl('Gateway Publishable Key'),
+            'text',
+            '5ccg4qbjvdA66pZBr4Pt2C4YEtFXmHtevE4zMLUZ3DE26yc5JrCarK5uF3MjGm5y',
+            xl('The public access key for secure tokenize of credit or debit card authorization. PCI compliance')
+        ),
+
+        'gateway_api_key' => array(
+            xl('Gateway API Login Auth Name or Secret'),
+            'text',
+            '8seGJk8A4E',
+            xl('The Auth Name or API key for selected account.Auth Name for Authorize.Net and API Secret for Stripe')
+        ),
+
+        'gateway_transaction_key' => array(
+            xl('Gateway Transaction Key'),
+            'text',
+            '9Peg2rE66FxP2p5p',
```

Comment:
Same question as to whether this is sensitive info?

Reply:
yes

Reply:
Makes sense to keep it backwards compatible for the original commit.
TODO for me
There is a new developer going through the globals and setting up the encrypt/decrypt and I'll include this on the list.